src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -50,6 +53,20 @@ public override void OnEventWritten(string name, object payload)
                     return;
                 }
 
+                try
+                {
+                    if (this.options.Filter?.Invoke(context) == false)
+                    {
+                        AspNetCoreInstrumentationEventSource.Log.RequestIsFilteredOut(Activity.Current.OperationName);
```

Comment:
nit: the log does not clearly indicates that the measurement is filtered out.

Reply:
I put a fix on #3982 let me know if that works!